ts/DataGrid/Accessibility/Accessibility.ts

```diff
@@ -116,45 +94,24 @@ class Accessibility {
     *
     * */
 
-    /**
-     * Load the accessibility options.
-     */
-    public loadOptions(): void {
-        const lang = this.dataGrid.options?.lang?.accessibility;
-        if (!lang) {
-            return;
-        }
-
-        this.editableCellDescriptionEl.textContent =
-            lang.cellEditing?.editable || '';
-        this.sortableColumnDescriptionEl.textContent =
-            lang.sorting?.sortable || '';
-    }
-
     /**
      * Add the description for the editable cell.
      *
      * @param cellElement
      * The cell element to add the description to.
      */
     public addEditableCellDescription(cellElement: HTMLElement): void {
-        cellElement.setAttribute(
-            'aria-describedby',
-            Accessibility.decriptionElementIds.editableCell
-        );
-    }
+        const description =
+            this.dataGrid.options?.lang?.accessibility?.cellEditing?.editable;
```

Comment:
We should keep the consistency of naming. 

I would prefer to description in API too.